README.md

```diff
@@ -276,6 +276,25 @@ file, rename and edit it.
 <?php
 
 $settings = array (
+    // Security settings that must be set to avoid insecure SHA-1 usage.
```

Comment:
Good idea, there is already a security warning section about the strict parameter, added SHA1 info there.

Basic settings already contain `baseurl` and `strict` so I think it is fine to also include the signature and digest algorithm there?